insights/specs/insights_archive.py

```diff
@@ -15,7 +15,7 @@ class InsightsArchiveSpecs(Specs):
     bios_uuid = simple_file("insights_commands/dmidecode_-s_system-uuid")
     blkid = simple_file("insights_commands/blkid_-c_.dev.null")
     brctl_show = simple_file("insights_commands/brctl_show")
-    ceph_df_detail = simple_file("insights_commands/ceph_df_detail_-f_json-pretty")
+    ceph_df_detail = simple_file("insights_commands/ceph_df_detail_-f_json")
     ceph_health_detail = simple_file("insights_commands/ceph_health_detail_-f_json-pretty")
     ceph_insights = simple_file("insights_commands/python_-m_insights.tools.cat_--no-header_ceph_insights")
     ceph_osd_df = simple_file("insights_commands/ceph_osd_df_-f_json-pretty")
```

Comment:
@psachin thanks for making all of these changes.  Since the commands changed you'll also need to change all of the archive specs to match.  Also since we need to be able to read archives created with the previous versions of  the commands you'll need to make the archive specs `first_file` with the old archive filename and the new archive filename.

Reply:
@bfahr Do you mean like an example below:

```
ceph_df_detail = first_file(["insights_commands/ceph_df_detail_-f_json-pretty", "insights_commands/ceph_df_detail_-f_json"])
```

Reply:
@bfahr I've implemented the `first_file` to work with old as well as new archives. Please check.